CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## 1.88.1-dev
 
+* **Potentially breaking bug fix:** Throw an error when passing a function or
+  mixin from one compilation to another.
+
```

Comment:
@nex3 Looks like 1.88.0 was never released?

Reply:
Huh, I guess I forgot to kick off a release. Thanks for noticing.